ydb/src/client_topic/list_types.rs

```diff
@@ -160,14 +160,16 @@ impl From<AlterConsumer> for RawAlterConsumer {
 #[derive(Debug, Clone, Default)]
 pub struct PartitioningSettings {
     pub min_active_partitions: i64,
-    pub partition_count_limit: i64,
+    pub max_active_partitions: i64,
+    pub auto_partitioning_settings: Option<ydb_grpc::ydb_proto::topic::AutoPartitioningSettings>,
```

Comment:
Extract change api, internal logic and etc to other PR please. For separate reformats/style updates/dependency update from api/logic updates. It will be much easer for research changes reason in the future.

Reply:
many of style fixes already done within upgrade rust version for the CI https://github.com/ydb-platform/ydb-rs-sdk/pull/356

Reply:
What about split the PR to 3 parts?
1. Add vulnerabilities linters and upgrade dependencies, MSRV
2. Fix api fields
3. Fix style (if need)

Reply:
Agreed will do it